src/Workleap.OpenApi.MSBuild/ProcessWrapper.cs

```diff
@@ -6,18 +6,20 @@ namespace Workleap.OpenApi.MSBuild;
 internal sealed class ProcessWrapper : IProcessWrapper
 {
     private readonly string _workingDirectory;
+    private readonly Dictionary<string, string?> _defaultEnvVars = new();
```

Comment:
This one can be static

```suggestion
    private static readonly Dictionary<string, string?> _defaultEnvVars = new();
```

Reply:
True!